Sources/AltSwiftUI/Source/Views/ReadOnly/Label.swift

```diff
@@ -0,0 +1,115 @@
+        self.title = title()
+        self.icon = icon()
+    }
+
+    public var body: View {
+        self
+    }
+}
+
+extension Label where Title == Text, Icon == Image {
+    /// Creates an instance that disaly an `icon` with a `title`.
+    ///
+    /// - Parameters:
+    ///     - title: The string of right part text of the label
+    ///     - icon: The visual representation of left part image of the label
+    public init<S: StringProtocol>(_ title: S, image: String) {
+        self.title = Text(title)
+        self.icon = Image(image)
+    }
+    
+    /// Creates an instance that disaly an `icon` with a `title`.
+    ///
+    /// - Parameters:
+    ///     - title: The LocalizedStringKey of right part text of the label
+    ///     - icon: The visual representation of left part image of the label
+    @available(iOS 14.0, *)
+    public init(_ title: LocalizedStringKey, image: String) {
+        self.title = Text(NSLocalizedString(title, comment: ""))
+        self.icon = Image(image)
+    }
+    
+    /// Creates an instance that disaly an `icon` with a `title`.
+    ///
+    /// - Parameters:
+    ///     - title: The string of right part text of the label
+    ///     - icon: The system icon name of left part image of the label
+    @available(iOS 14.0, *)
+    public init<S: StringProtocol>(_ title: S, systemImage: String) {
+        self.title = Text(title)
+        self.icon = Image(uiImage: UIImage(systemName: systemImage) ?? UIImage())
+    }
+    
+    /// Creates an instance that disaly an `icon` with a `title`.
+    ///
+    /// - Parameters:
+    ///     - title: The LocalizedStringKey of right part text of the label
+    ///     - icon: The system icon name of left part image of the label
+    @available(iOS 14.0, *)
+    public init(_ title: LocalizedStringKey, systemImage: String) {
+        self.title = Text(NSLocalizedString(title, comment: ""))
+        self.icon = Image(uiImage: UIImage(systemName: systemImage) ?? UIImage())
+    }
+}
+
+extension Label: Renderable {
+    public func updateView(_ view: UIView, context: Context) {
+        guard let stackView = view as? SwiftUIStackView else { return }
+        setupView(stackView, context: context)
+    }
+    
+    public func createView(context: Context) -> UIView {
+        let hstack = HStack {
+            Spacer()
+                .frame(height: 0)
+            self.icon
+            self.title
+            Spacer()
+                .frame(height: 0)
+        }
+        if let stackView = hstack.createView(context: context) as? SwiftUIStackView {
+            setupView(stackView, context: context)
+            return stackView
+        }
+        
+        return UIView()
+    }
+    
+    private func setupView(_ view: UIStackView, context: Context) {
+        guard let labelStyleType = context.viewValues?.labelStyle?.labelStyleType else { return }
+        
+        context.viewOperationQueue.addOperation {
```

Comment:
It's not necessary to use  the operation queue in this case. 

Note: The operation queue main purpose is to flatten view hierarchy updates that would add strain to the call stack causing potential stack overflows. Usually this is used when you want to call `updateView` of children views.